README.md

```diff
@@ -80,6 +80,34 @@ Integration with the Genius Yield DEX has never been easier.
 > [!TIP]
 > Have a look at sample configuration in [Building locally from source using the Haskell Toolchain](#building-locally-from-source-using-the-haskell-toolchain) section for thorough explanation of options made available to configure the server.
 
+### Setting up the API Server with Docker (Kupo)
+
+If you're looking to utilize the API server alongside the Kupo provider backend, you're in luck! We've streamlined the process for you.
+
+Inside the repository, you'll find a `docker-compose-kupo.yml` file, which serves as a blueprint for running a local Cardano node, Kupo, and the API server seamlessly.
+
+```bash
+# Step 1: Clone the repository
+git clone git@github.com:geniusyield/dex-contracts-api.git
+
+# Step 2: Initialize and update submodules
+cd dex-contracts-api
+git submodule init
+git submodule update
```

Comment:
Should we checkout a particular commit of submodule to prevent for any surprises?

Reply:
I have added an explicit revision in: 9b2883038e50b4ed2cc2fb31b083f84a48fbeeb6